src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -73,24 +70,24 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
           || (currentAcquisitionPhase.equals(AcquisitionPhase.RECONCILING_TASKS)
-              && !isNameNode1Initialized())
+          && !isNameNode1Initialized())
           || (status.getMessage().equals(HDFSConstants.NAME_NODE_BOOTSTRAP_MESSAGE)
-              && !isNameNode1Initialized())) {
+          && !isNameNode1Initialized())) {
```

Comment:
These look wrong to me. The &&'s are nested within the previous line's (), so I think it reads better for them to be indented to match.